### PR TITLE
[10.0] web_tree_image: Porting to 10.0

### PR DESCRIPTION
**Pull request by danieltorres7**
- Changed model to installable.
- Tested and everything seems ok.

### CODE REVIEW

**Comment by danieltorres7** [comment]
@leorochael, can you take a look?

**Comment by pedrobaeza** [comment]
You should switch to new JS API

**Comment by leorochael** [comment]
@pedrobaeza Can you give us some pointers? Where can we find docs or examples of the new JS API?

**Comment by pedrobaeza** [comment]
I'm afraid there isn't. You need to see sample in OCA/web repository for 9.0 and odoo itself.

**Comment by leorochael** [comment]
@pedrobaeza, if it's a v9 new JS API, isn't the changes landed in #290, enough?

**Comment by pedrobaeza** [comment]
Yeah, indeed, sorry for the noise. I though it was a direct migration from 8.0. Then it's enough.

File: web_tree_image/__manifest__.py [hunk 1]
```diff
@@ -27,7 +27,7 @@
 ##############################################################################
 {
     'name': 'Show images in tree views',
-    'version': '9.0.1.1.0',
```
Comment (leorochael) [inline web_tree_image/__manifest__.py:30]: Please merge this commit with the previous one

**Review by leorochael — CHANGES_REQUESTED** [review]

File: web_tree_image/static/src/js/web_tree_image.js [hunk 2]
```diff
@@ -19,7 +19,7 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-odoo.define('web.tree_image', function (require) {
+odoo.define('web.WebTreeImage', function (require) {
```
Comment (leorochael) [inline web_tree_image/static/src/js/web_tree_image.js:22]: @pedrobaeza @StefanRijnhart, What do you think of this change?

It looks ok to me, though it doesn't seem strictly necessary to port to v10

**Review by leorochael — COMMENTED** [review]
LGTM :+1:

File: web_tree_image/static/src/js/web_tree_image.js (hunk 2 above)
Comment (pedrobaeza) [reply web_tree_image/static/src/js/web_tree_image.js:22]: This only impacts in possible module that inherits this, so no actual problem I think.

**Review by StefanRijnhart — APPROVED** [review]
All good, thank you!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**